tests/test_markitdown.py

```diff
@@ -7,7 +7,8 @@
 import requests
 
 from warnings import catch_warnings, resetwarnings
-
+import sys
+sys.path.insert(0, "/home/yxl/Projects/markitdown/src")
```

Comment:
I think don't need this. Let me know if you need help setting up the test!

Reply:
I will remove this, it won't be used